src/components/TopNavigation.jsx

```diff
@@ -0,0 +1,51 @@
+import logo from "../assets/website-logo.png";
+import { Link } from "react-router-dom";
+import "../styles/TopNavigation.css";
+
```

Comment:
If you separate the module import with local import will increase the readability
Ex:
```
//All module imports at the start
import React from "react";

//All local import
import "./landingPage.css"
```

Reply:
Thanks, nice suggestion!

Reply:
Done